README.md

```diff
@@ -5,7 +5,7 @@
 Podman (the POD MANager) is a tool for managing containers and images, volumes mounted into those containers, and pods made from groups of containers.
 Podman is based on libpod, a library for container lifecycle management that is also contained in this repository. The libpod library provides APIs for managing containers, pods, container images, and volumes.
 
-* [Latest Version: 3.4.0](https://github.com/containers/podman/releases/latest)
+* [Latest Version: 4.1.0-dev](https://github.com/containers/podman/releases/latest)
```

Comment:
You can leave this alone, this generally points to latest stable

Reply:
@mheon , @TomSweeneyRedHat added a comment in version.go to update this as well...

Gentlemen, should we just use `[Latest Version](...) `and skip the explicit version number?

Reply:
I pref the latest version, but yeah, it would be nice if we could make sure to update this in a more timely manner.